python/paddle/fluid/tests/unittests/mkldnn/test_fusion_lstm_bf16_mkldnn_op.py

```diff
@@ -32,7 +32,8 @@ def set_confs(self):
     def test_check_output(self):
         for use_seq in {True, False}:
             self.attrs['use_seq'] = use_seq
-            self.check_output(check_dygraph=False, no_check_set=["Cell"])
+            self.check_output(
+                check_dygraph=False, no_check_set=["Cell"], atol=2e-2)
```

Comment:
In file `python/paddle/fluid/tests/unittests/op_test.py`, `atol = 0.03` is not a good way to check forward accuracy. This PR modified the relative error of checking the accuracy of `bfload16` data type and deleted the limitation of 0.03. And add `atol = 2e-2` here to keep the same accuracy limit as before to ensure the test pass.